.github/workflows/renovate-vault.yml

```
@@ -12,18 +12,20 @@ on:
         required: false
         default: "false"
         type: string
-  # Run twice in the early morning (UTC) for initial and follow up steps (create pull request and merge)
   schedule:
-    - cron: '30 4,6 * * *'
+    # For main branche, runs twice every other day in the first two
```

```suggestion
    # For main branches, runs twice every other day in the first two
```
Small typo, otherwise LGTM !